war/src/main/js/components/tooltips/index.js

```diff
@@ -19,16 +19,18 @@ function registerTooltip(element) {
     element._tippy.destroy();
   }
 
+  const tooltip = element.getAttribute("tooltip");
+  const htmlTooltip = element.getAttribute("data-html-tooltip");
   if (
-    element.hasAttribute("tooltip") &&
-    !element.hasAttribute("data-html-tooltip")
+    tooltip != null &&
+    tooltip.length > 0 &&
+    (htmlTooltip == null || htmlTooltip.length == 0)
```

Comment:
Are you doing `null` checks deliberately not with `===` etc? (Curiosity more than request for change)